src/tests/ftest/unittest/Unittest.py

```diff
@@ -25,65 +25,79 @@
 
 from avocado.utils import process
 from general_utils import get_file_path
-from apricot import Test
+from apricot import Test, TestWithServers
 
-class UnitTest(Test):
+def unittest_runner(self, unit_testname):
     """
-    Avocado Unit Test class.
+    Common unitetest runner function.
+    Args:
+        unit_testname: unittest name.
+    return:
+        None
+    """
+    name = self.params.get("testname", '/run/UnitTest/{0}/'
+                           .format(unit_testname))
+    bin_path = get_file_path(name, "install/bin")
+
+    cmd = ("{0}".format(bin_path[0]))
+    return_code = process.system(cmd)
+    if return_code is not 0:
+        self.fail("{0} unittest failed with return code={1}.\n"
+                  .format(unit_testname, return_code))
+
+class UnitTestWithoutServers(Test):
+    """
+    Avocado Unit Test class for tests which don't need servers.
```

Comment:
According to the global validation test plan the format for this docstring should be:

```
"""
Test Class Description:
<description>
:avocado: recursive
"""
```

Also the docstrings for the test_* methods should be:

```
"""
Jira ID: <id>
Test Description: <description>
Use Cases: <use cases>
:avocado: tags=<tags>
"""
```

Reply:
Done. Except Jira ID for already present unit test. Was unable to find the old Jira ticket under which these tests were added to avocado framework. Not sure if these tests were added as part of a specific ticket.